### PR TITLE
RATEST-92:Failing Test: Contact Info Test

### PR DESCRIPTION
**Pull request by sherrif10**
https://issues.openmrs.org/browse/RATEST-92    cc @k-joseph  @kdaud

### CODE REVIEW

File: ui-tests/src/test/java/org/openmrs/reference/ContactInfoTest.java [hunk 1]
```diff
@@ -35,6 +36,7 @@ public void setUp() throws Exception {
     }
 
     @Test
+    @Ignore
```
Comment (kdaud) [inline ui-tests/src/test/java/org/openmrs/reference/ContactInfoTest.java]: Do you need `@Ignore` annotation on the test method ?

File: ui-tests/src/test/java/org/openmrs/reference/page/RegistrationEditSectionPage.java [hunk 2]
```diff
@@ -23,6 +25,7 @@
     private static final By STATE_PROVINCE = By.id("stateProvince");
     private static final By COUNTRY = By.id("country");
     private static final By POSTAL_CODE = By.id("postalCode");
+	private static final String str = "Must be a valid phone number (with +, -, numbers or parentheses))";
```
Comment (kdaud) [inline ui-tests/src/test/java/org/openmrs/reference/page/RegistrationEditSectionPage.java:28]: Indention here

File: ui-tests/src/test/java/org/openmrs/reference/page/RegistrationEditSectionPage.java [hunk 3]
```diff
@@ -115,12 +118,15 @@ public void enterPhoneNumber(String phone) {
 
     public String getInvalidPhoneNumberNotification() {
         List<String> errors = getValidationErrors();
+        if(errors !=null) {
```
Comment (kdaud) [inline ui-tests/src/test/java/org/openmrs/reference/page/RegistrationEditSectionPage.java:121]: Indention here

File: ui-tests/src/test/java/org/openmrs/reference/page/RegistrationEditSectionPage.java [hunk 4]
```diff
@@ -5,6 +5,8 @@
 import org.openmrs.uitestframework.page.Page;
 import org.openqa.selenium.By;
 
+import com.google.common.base.Optional;
```
Comment (kdaud) [inline ui-tests/src/test/java/org/openmrs/reference/page/RegistrationEditSectionPage.java:8]: Is this import neccessary?

File: ui-tests/src/test/java/org/openmrs/reference/ContactInfoTest.java [hunk 5]
```diff
@@ -65,6 +67,7 @@ public void contactInfoTest() throws Exception {
         patientDashboardPage = activeVisitsPage.goToPatientDashboardOfLastActiveVisit();
         patientDashboardPage.clickOnShowContact();
         assertTrue(patientDashboardPage.getTelephoneNumber().contains("111111111"));
+        patientDashboardPage.waitForPage();
```
Comment (kdaud) [inline ui-tests/src/test/java/org/openmrs/reference/ContactInfoTest.java:70]: Any reason for adding waiter here?

File: ui-tests/src/test/java/org/openmrs/reference/page/RegistrationEditSectionPage.java (hunk 4 above)
Comment (sherrif10) [reply ui-tests/src/test/java/org/openmrs/reference/page/RegistrationEditSectionPage.java]: Resolved

File: ui-tests/src/test/java/org/openmrs/reference/ContactInfoTest.java (hunk 1 above)
Comment (sherrif10) [reply ui-tests/src/test/java/org/openmrs/reference/ContactInfoTest.java]: Sorry my bad

File: ui-tests/src/test/java/org/openmrs/reference/ContactInfoTest.java (hunk 5 above)
Comment (sherrif10) [reply ui-tests/src/test/java/org/openmrs/reference/ContactInfoTest.java]: The reason behind this waiter is that patientDashboardPage is taking long to be executed fully, i tested this on two instances , the running host will quit with that waiter, but on qa server it really takes long , thats why i added that waiter

**Pull request closed without merging**